packages/@react-types/table/src/index.d.ts

```diff
@@ -130,8 +130,13 @@ export type CellElement = ReactElement<CellProps>;
 export type CellRenderer = (columnKey: Key) => CellElement;
 
 export interface TableCollection<T> extends GridCollection<T> {
+  // TODO perhaps elaborate on this? maybe not clear enought, essentially returns the table header rows (e.g. in a tiered headers table, will return the nodes containing the top tier column, next tier, etc)
```

Comment:
Should we remove this TODO and make it a story?

Reply:
It already is covered in all the stories since every table has headers, this is more of a question of the typescript definition comment